packages/x-components/src/__tests__/utils.ts

```diff
@@ -22,17 +21,10 @@ import { MutationsDictionary } from '../store/mutations.types';
 import { RootXStoreState, XStoreModule } from '../store/store.types';
 import { cleanGettersProxyCache } from '../store/utils/getters-proxy.utils';
 import { ExtractState, XModule, XModuleName } from '../x-modules/x-modules.types';
-import { PlatformAdapterDummy, SearchAdapterDummy } from './adapter.dummy';
+import { SearchAdapterDummy } from './adapter.dummy';
 import Mock = jest.Mock;
 
 export type MockedSearchAdapter = {
```

Comment:
What would you think of renaming SearchAdapter in the variable names by PlatformAdapter to avoid confusion with the old  `@empathyco/x-adapter#SearchAdapter`?